flink-metrics/flink-metrics-datadog/src/main/java/org/apache/flink/metrics/datadog/metric/MetricType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.metrics.datadog.metric;
+
+/**
+ * Metric types supported by Datadog
+ * */
+public enum MetricType {
+	gauge, counter
```

Comment:
yes, 'counter' and 'gauge' are specific types in Datadog, and these will be fields `"type":"gauge"` and `"type":"counter"` in json sent to Datadog. 

Since they are meant for serialization mentioned above, I don't see any benefits make them upper-case, or doing something like `GAUGE("gauge"), COUNTER("counter")`